src/ansys/aedt/core/generic/general_methods.py

```diff
@@ -732,14 +727,10 @@ def generate_unique_name(root_name, suffix="", n=6):
         Newly generated name.
     """
     alphabet = string.ascii_uppercase + string.digits
-    if is_ironpython:
-        import random
 
-        uName = "".join(random.choice(alphabet) for _ in range(n))  # nosec B311
-    else:
-        import secrets
+    import secrets
 
-        uName = "".join(secrets.choice(alphabet) for _ in range(n))
+    uName = "".join(secrets.choice(alphabet) for _ in range(n))
 
     unique_name = root_name + "_" + uName
```

Comment:
```suggestion
    name = "".join(secrets.choice(alphabet) for _ in range(n))

    unique_name = root_name + "_" + name
```

Reply:
Just taking this review as an opportunity to apply snake case when I see it needed :)